lib/generators/curation_concerns/install_generator.rb

```diff
@@ -101,8 +101,11 @@ def resource_types_config
       copy_file "config/authorities/resource_types.yml", "config/authorities/resource_types.yml"
     end
 
-    def simple_form_bootstrap
-      generate 'simple_form:install --bootstrap'
```

Comment:
I am okay with either option. @jcoyne asked me to remove this line, so what is the majority opinion? @mjgiarlo, @cam156, etc?